### ISSUE TITLE
Location header field in JAX-RS response is set twice

### ISSUE THREAD

**Issue by RovoMe** [restlet/restlet-framework-java#948]
On returning a `Response` object for a created resource (usually from a HTTP POST or PUT request) in a JAX-RS service where the response is created like this
```java
return Response.created(URI.create(someUri)).build();
```
which is basically a shortcut for
```java
return Response.status(Response.Status.CREATED).location(URI.create(someUri)).build();
```
the HTTP header of the received response contains multiple `Location` header fields.

To be a bit more specific:
Imagine I have a JAX-RS product CRUD service running at `https://localhost:9876/rest/products` like the following:
```java
@Path("/rest/products")
public ProductResource 
{
    @POST
    @Consumes(MediaType.APPLICATION_JSON)
    public Response addProduct(InputStream is) 
    {
        Product product = convertJsonToProduct(is);
        product.setId(1234);
        return Response.created(URI.create("/rest/products/"+product.getId())).build();
    }
    ...
}
```
where 1234 is the unique ID of the created resource, the header will contain two location-headers:
`Location: https://localhost:9876/rest/products/1234`
`Location: /rest/products/1234`
adding f.e. `98765` to `product.getId()` in the sample code above will change both location header fields to:
`Location: https://localhost:9876/rest/products/123498765`
`Location: /rest/products/123498765`

This behavior is however not only related to `Response.Status.CREATED` responses but to all responses that set `.location(...)`.

Why is the location header field set twice in the response header? I' currently running version 2.2.2 of Restlet and run a Jetty based Restlet environment with Spring and JAX-RS support.

**Comment by thboileau** [comment]
may be linked with a bug detected while handling this issue #799.

**Comment by thboileau** [comment]
Thanks for reporting this issue. it has been fixed and will be available in 2.3.2.
